extensions/meme-generator/package.json

```diff
@@ -38,26 +38,26 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.40.1",
-    "node-fetch": "^3.2.10",
-    "run-applescript": "^6.0.0",
-    "tempy": "^3.0.0"
+    "@raycast/api": "^1.81.2",
+    "@raycast/utils": "^1.16.3",
+    "node-fetch": "^3.3.2",
+    "run-applescript": "^7.0.0",
```

Comment:
(non-blocking) You shouldn't need this extra dependency now. Please check (runAppleScript)[https://developers.raycast.com/utilities/functions/runapplescript] and [Clipboard.copy](https://developers.raycast.com/api-reference/clipboard) `file` options.